Yogesh_Ghodmare/HTML/Assig6/Que.1.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Form</title>
+        <link rel="stylesheet" href="Form_effect.css" />
+    </head>
+    <body>
+        <div>
+            <header>
+                <h2>MI-9 Intern HTML Assesment</h2>
+            </header>
+            <br /><br />
+            <hr />
+            <h3><i>Presonel Details</i></h3>
+            <form action="">
+                <label for="fname">Fist Name<input type="text" placeholder="Fist Name" /></label>
+                <br /><br />
+                <label for="lname">Last Name<input type="text" placeholder="Last Name" /></label><br /><br />
```

Comment:
placeholder second word first letter should be small

Reply:
Changes Done